### ISSUE TITLE
Heroku deployment failure

### ISSUE THREAD

**Issue by matheswarwan** [salesforce-marketingcloud/sfmc-example-jb-custom-activity#25]
Yargs-parser artifact missing while deploying in heroku.

> npm ERR! code ENOTFOUND
> npm ERR! syscall getaddrinfo
> npm ERR! errno ENOTFOUND
> npm ERR! network request to https://artifactory-dev.exacttarget.com:443/artifactory/api/npm/sfmc-npm/yargs-parser/-/yargs-parser-13.1.2.tgz failed, reason: getaddrinfo ENOTFOUND artifactory-dev.exacttarget.com

**Comment by rmccu** [comment]
I have the same problem. Can this be escalated?

**Comment by rmccu** [comment]
Note, I tried both Heroku and running locally on my machine. This looks like the specified resource is no longer available.

**Comment by mgalvao2012** [comment]
I'm having the same issue. Is there any workaround to fix it?

**Comment by JoeyNovais** [comment]
Same issue here, if anyone has resolved it please share !

**Comment by JoeyNovais** [comment]
TL;DR
use node v14.20.1 or any that uses npm 6.x.x

Ok guys, if you experience this: 
![image](https://user-images.githubusercontent.com/2325059/195357659-ea27793a-1aac-449e-89da-e1b927680313.png)
The important part is not the error, but the warning. It tries to convert and download again from the artifactory thus the error. To avoid the conversion you'll need to use a lower version of npm compatible with the current lockfile, so npm version 6.x.x

After changing my npm version it works fine !

**Comment by JoeyNovais** [comment]
In case you want to deploy through Heroku, just add those lines in your package.json
`"engines":{
    "node": "14.20.1",
    "npm": "6.14.17"
  }`

@rmccu @mgalvao2012
